src/workspace.js

```diff
@@ -1268,6 +1268,7 @@ module.exports = class Workspace extends Model {
 
   handleGrammarUsed (grammar) {
     if (grammar == null) { return }
+    this.packageManager.triggerActivationHook(`${grammar.scopeName}:root-scope-used`)
     return this.packageManager.triggerActivationHook(`${grammar.packageName}:grammar-used`)
```

Comment:
Outside the scope of this PR, but remarking on it anyway; should we :fire the `return` here? There is no value returned by `triggerActivationHook`, so returning here doesn't make any sense and is simply confusing.

Reply:
I'd remove it. Should be fine to do it in this PR right?

Reply:
Yeah, definitely fine in my opinion. Just didn't want to expand the scope of your PR :sweat_smile: